app/views/posts/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>List of posts here</h1>
\ No newline at end of file
+<div class="container my-3">
+  <%= render partial: 'shared/user', locals: { user: @user } %>
+  <%= render partial: 'shared/bio', locals: { user: @user } %>
+
+  <% @posts.each do |post| %>
```

Comment:
* Kindly use the `post.recent_comments` to render 5 most recent comments as shown in the wireframe provided 
![image](https://github.com/anderCM/blog-app/assets/52497246/3e813100-b97a-4cb6-b8a3-ae25834cfc63)

Reply:
@Davidosky007  Thanks I noticed that I have already created the method `last_5_comments`. I will reuse it meet the requirements